### PR TITLE
Portal updates and general bug fixes

### PR DESCRIPTION
**Pull request by sjpadgett**
- uninitialized  temp global translations
- fix long CCDA buffer overrun ccdaservice.
- fix logger class use not in scope
- fix logging out
- portal lab result list fix.
- fix divtoggle() encounter reports expand/collapse
- upgrade all jquery lib from 1-11-3 to current
- removed all signer modals replaced with dynamic  template
- cleanup and rewrite signer feature.

### CODE REVIEW

**Comment by sjpadgett** [comment]
@bradymiller I still need to do some xlate for js api. It's most all dynamic so needs some thought. Let me know if you need help with login issue

File: portal/lib/appsql.class.php [hunk 1]
```diff
@@ -26,6 +26,8 @@
  * @param
  *            wrapper class for moving some care coordination zend product
  */
+$ignoreAuth = true;
+require_once(dirname(__FILE__) . '/../../interface/globals.php');
```
Comment (bradymiller) [inline portal/lib/appsql.class.php]: hi @sjpadgett  , This makes me very nervous. I think this means that any script that includes this script would be able to skip authentication.

Comment (sjpadgett) [reply portal/lib/appsql.class.php]: I forgot it was there. when you added logger class it was out of scope with auto loader so I meant to go back and include the class direct.

Comment (bradymiller) [reply portal/lib/appsql.class.php]: doh, sorry about that,
Looks like a couple places that use this script need classes for autoload. We should look at those places to see if either can get global.php included safely or, if not, then just do the autoload by itself.

Comment (sjpadgett) [reply portal/lib/appsql.class.php]: already fixed and will put up in a minute.

**Comment by sjpadgett** [comment]
am going to leave the three alerts un translated for now. I need this api for phase two when I allow any device on intranet/same-origin to act as kiosk and popup signature pad for patient signature. Will be neatO:) Will see about translations then

**Comment by bradymiller** [comment]
hi @sjpadgett , sounds good. code looks good to me. -brady

**Comment by sjpadgett** [comment]
bringing in in case you need to mod portal and bug fixes. Thanks for checking me.
Next version will be that any form field with an input or anchor(think LBF) having attr's of say, data-signer='getandplace' data-signer='anychannel', will send request to popup a signature pad on selected device. Most likely a tablet. 
also plan to add a digital signing to transaction for a digital signature making it legal.

**Pull request merged**